packages/levenshtein/levenshtein.1.1.3/opam

```diff
@@ -0,0 +1,15 @@
+opam-version: "1.2"
+version: "1.1.3"
+maintainer: "jun.furuse@gmail.com"
+authors: "Jun Furuse"
+homepage: "https://bitbucket.org/camlspotter/ocaml_levenshtein/"
+bug-reports:
+  "https://bitbucket.org/camlspotter/ocaml_levenshtein/issues?status=new&status=open"
+dev-repo: "hg://https://bitbucket.org/camlspotter/ocaml_levenshtein"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+depends: [
+  "ocamlfind" {build}
+  "jbuilder" {build}
+  "ppx_test" {>= "1.6.0"}
+]
+available: [ocaml-version >= "4.02.1"]
```

Comment:
Does jbuilder even support `4.02.1`? Didn't some key ppx details change between `4.02.1` and `4.02.3`? Seems unlikely that you would support both between ppx_test.

Reply:
@camlspotter have you tested a build on `4.02.1`?

Reply:
Probably not worth delaying the release for this. Nobody uses 4.02.1 and the constraint should really go on ppx_test anyway.